code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -83,15 +85,39 @@
 /mob/living/simple_animal/construct/cultify()
 	return
 
-/mob/living/simple_animal/construct/New()
+/mob/living/simple_animal/construct/New(var/mob/living/carbon/user)
 	..()
 	hud_list[CONSTRUCT_HUD] = image('icons/mob/hud.dmi', src, "consthealth100")
-	add_language(LANGUAGE_CULT)
-	default_language = all_languages[LANGUAGE_CULT]
 	for(var/spell in construct_spells)
 		src.add_spell(new spell, "cult_spell_ready", /obj/abstract/screen/movable/spell_master/bloodcult)
+	setup_type(user)
```

Comment:
The issue with this is that you're passing user as the first parameter to `New`.
`New` expects the first parameter to be the location in which the atom is created.
The reason why I told you to move this into a proc was so that you could call it after `New`, avoiding messing with `New`'s arguments at all. You *can* keep the user argument here, as long as you make it not the first argument, but it makes more sense to me to make it a separate step.

Reply:
The reason why I kept the user argument in new() was because the setup of the color and construct languages needs to know who was creating the construct. I'm not sure how I would get that information without the user argument in new()

Reply:
Like I said, either call the proc from the outside, after you have `new`'d the construct, or make `user` the second argument.

Reply:
done, dunno why i didnt think to call the proc from the outside